canonical-web-and-design/snapcraft.io
# Remove legacy brand store files

## Done
Removed the legacy templates and JS files for the previous version of the brand store

## QA
- Go to https://snapcraft-io-3730.demos.haus/admin
- Check that the brand store is still there

## Issue
Fixes #3720
